### PR TITLE
Implementing employee screens

### PR DESCRIPTION
**Pull request by r176378**
implemented screens employee and employees. many smaller changes.

### CODE REVIEW

**Review by RenanEug — APPROVED** [review]
Approved.

File: src/main/java/Controllers/DataController.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            session.close();
+        } catch (Exception exception) {
+            exception.printStackTrace();
+        }
+    }
+
+    public Object getObjectById(Class data,  Integer id) {
+        Object retrievedObject = null;
+        try {
+
```
Comment (kaueel) [inline src/main/java/Controllers/DataController.java:77]: unecessary new line

File: src/main/java/Controllers/MainScreensController.java [hunk 2]
```diff
@@ -45,18 +42,19 @@ public static synchronized MainScreensController getInstance() {
...
-    //Avoids rebiulding of already used screens, saving bult panes in a hashMap
+    public String getTitle() {
+        return currentScreenTitle;
+    }
+
     private Pane getScreenFromMap(String fxmlPath) {
-        Pane pane = screensMap.get(fxmlPath);
-        if (pane == null) {
+        Pane pane = null;
+
```
Comment (kaueel) [inline src/main/java/Controllers/MainScreensController.java]: I guess u need to fix tabs here

File: src/main/java/Controllers/Screen.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package Controllers;
+
+public class Screen {
```
Comment (kaueel) [inline src/main/java/Controllers/Screen.java:3]: Please, review this class to be an interface

**Review by kaueel — CHANGES_REQUESTED** [review]

**Review by kaueel — APPROVED** [review]

**Pull request merged**